tests/core/normalization/test_default_outbound_normalization.py

```diff
@@ -1,11 +1,129 @@
+from typing import (
+    Any,
+)
+
 import pytest
 
 from eth_tester.normalization import (
     DefaultNormalizer,
 )
 from tests.utils import (
+    make_blob_txn,
+    make_block,
+    make_log,
     make_receipt,
+    make_withdrawal,
+)
+
+
+@pytest.mark.parametrize(
+    "value,expected",
+    (
+        (b"\x01" * 20, f"0x{'01' * 20}"),
+        (f"0x{'01' * 20}", f"0x{'01' * 20}"),
+        (b"\x01" * 32, None),
+        (b"\x01" * 21, None),
+        (b"\x01" * 19, None),
+        (b"\x01" * 0, None),
+        (b"\x01" * 1, None),
+    ),
+)
+def test_outbound_account_normalization(value: Any, expected: Any) -> None:
+    if expected:
```

Comment:
We've been moving away from tests that have an if statement in them because it allows the test naming around the expectation to be clearer. What do you think about splitting this into 2 tests - maybe something like `test_valid_outbound_account_normalization` and `test_invalid_outbound...`. Feel free to take or leave!

Reply:
Definitely didnt consider that, I'll remember to do that going forward. I'll update these and I also see some other tests that could be simpler if I split them into separate tests.